JavaScript/packages/recognizers-number/src/number/french/extractors.ts

```diff
@@ -40,12 +40,7 @@ export class FrenchNumberExtractor extends BaseNumberExtractor {
         // Add filter
         let ambiguityFiltersDict = new Array<RegExpRegExp>();
 
-        if (mode != NumberMode.Unit){
-
-            for (let [ key, value ] of BaseNumbers.AmbiguityFiltersDict){
-                ambiguityFiltersDict.push({ regExpKey: RegExpUtility.getSafeRegExp(key, "gs"), regExpValue: RegExpUtility.getSafeRegExp(value, "gs")})
-            }
-            
+        if (mode != NumberMode.Unit){            
```

Comment:
Nitpick: Can you fix the code standard and add the whitespaces between ) and { ?